mm2src/coins/lightning/ln_events.rs

```diff
@@ -524,14 +560,23 @@ impl LightningEventHandler {
         );
 
         let db = self.db.clone();
+        let trusted_nodes = self.trusted_nodes.clone();
         let channel_manager = self.channel_manager.clone();
         let platform = self.platform.clone();
         spawn(async move {
             if let Ok(last_channel_rpc_id) = db.get_last_channel_rpc_id().await.error_log_passthrough() {
                 let user_channel_id = last_channel_rpc_id as u64 + 1;
-                if channel_manager
-                    .accept_inbound_channel(&temporary_channel_id, user_channel_id)
-                    .is_ok()
+                if (trusted_nodes.lock().contains(&counterparty_node_id)
+                    && channel_manager
+                        .accept_inbound_channel_from_trusted_peer_0conf(
+                            &temporary_channel_id,
+                            &counterparty_node_id,
+                            user_channel_id,
+                        )
+                        .is_ok())
```

Comment:
`accept_inbound_channel_from_trusted_peer_0conf` should only be called if `is_node_trusted` is true, otherwise `accept_inbound_channel` should be called.

It can be done like this
``` rust
let trusted_nodes = trusted_nodes.lock().clone();
let accepted_inbound_channel_with_0conf = trusted_nodes.contains(&counterparty_node_id)
    && channel_manager
    .accept_inbound_channel_from_trusted_peer_0conf(
        &temporary_channel_id,
        &counterparty_node_id,
        user_channel_id,
    )
    .is_ok();

if accepted_inbound_channel_with_0conf
    || channel_manager
        .accept_inbound_channel(&temporary_channel_id, &counterparty_node_id, user_channel_id)
        .is_ok()
```

Reply:
Done